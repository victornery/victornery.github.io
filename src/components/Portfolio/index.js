import React from 'react'

const Portfolio = () => (
  <section>
    Seção Portfolio
  </section>
)

export default Portfolio