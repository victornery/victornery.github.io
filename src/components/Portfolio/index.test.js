import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Portfolio from './index'

it('render without crash', () => {
  const parent = document.createElement('main')
  ReactDOM.render(<Portfolio />, parent)
  ReactDOM.unmountComponentAtNode(parent)
})

it('render correctly', () => {
  const tree = renderer.create(<Portfolio />).toJSON()
  expect(tree).toMatchSnapshot()
})