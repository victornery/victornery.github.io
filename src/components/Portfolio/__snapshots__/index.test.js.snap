// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<section
  className="portfolio"
>
  Seção Portfolio
</section>
`;
